core/test/regression/site/site_spec.js

```diff
@@ -1915,7 +1915,7 @@ describe('Integration - Web - Site', function () {
                             response.statusCode.should.eql(200);
                             response.template.should.eql('tag');
 
-                            postSpy.args[0][0].options.filter.should.eql('(tags:\'bacon\'+tags.visibility:public)+page:false');
+                            postSpy.args[0][0].options.filter.should.eql('(tags:\'bacon\'+tags.visibility:public)+type:post');
```

Comment:
This bit got me very confused. Wasn't sure why the "frontend" was supposed to even know about the `type` here but in the end, it's just testing if the serializer created a correct default filter :man_facepalming: